script/changelog/parser.ts

```diff
@@ -35,20 +35,28 @@ function capitalized(str: string): string {
   return str.charAt(0).toUpperCase() + str.slice(1)
 }
 
-function getChangelogEntry(commit: IParsedCommit, pr: IAPIPR): string {
+export function findIssueRef(body: string): string {
   let issueRef = ''
-  let type = PlaceholderChangeType
-  const description = capitalized(pr.title)
 
-  const re = /Fixes #(\d+)/gi
+  const re = /(close[s]?|fix(e[sd])?|resolve[sd]):?\s*#(\d+)/gi
```

Comment:
The use of `(e[sd])` here means a second group will exist when matching `Fixes: #{id}` and I'm not sure of a better way to do this - but it's easy enough here to poke at "the last group" below so I'm somewhat okay with this.

Reply:
`(?:e[sd])` does the same thing as `(e[sd])`, except it doesn’t create a group number that you can access later.

Reply:
Nice shout. Actually, maybe I want to keep the group number around (so it's always three). Thinking on this some more...